stackslib/src/chainstate/burn/db/sortdb.rs

```diff
@@ -876,6 +876,16 @@ const SORTITION_DB_SCHEMA_8: &'static [&'static str] = &[
     r#"ALTER TABLE snapshots ADD miner_pk_hash TEXT DEFAULT NULL"#,
 ];
 
+const SORTITION_DB_SCHEMA_9: &'static [&'static str] = &[
```

Comment:
SortitionDB version 8 is already new in `next`.

Reply:
Yup, fixed